README.md

```diff
@@ -65,27 +65,35 @@ To get the bash shell, you must first activate the feature in Windows.
   * Restart if necessary
 2. Enable the Windows Subsystem for Linux feature
   * From Start, search for "Turn Windows features on or off" (type 'turn')
-  * Select Windows Subsystem for Linux (beta)
+  * Scroll down and Select Windows Subsystem for Linux (beta)
   * Click OK
   * Restart if necessary
-3. Complete Installation
-  * Open a cmd prompt and type "bash"
+3. Install a Linux Distribution
+  * Open the Microsoft Store (Win + type 'store') 
+  * Search for Ubuntu (Orange icon) or whichever you prefer (This doc will assume Ubuntu)
+  * Install the app
+4. Complete Installation
+  * Open a cmd prompt (Win key + type 'cmd')
+  * In the command prompt window type "bash"
   * Accept the license
   * Create a new UNIX user account (this is a separate account from your Windows account)
+5. Test it
+  * Run `sudo apt update`
 
-Next actions are performed in WSL Bash application and assumes that Ubuntu provider is used.
+Next actions are performed in WSL Bash application (cmd window -> `bash`) and assumes that Ubuntu provider is used.
 
 1. Install required libraries:
     ```
     sudo apt install libcurl4-openssl-dev
     sudo apt install libboost-all-dev
+    sudo apt install g++
```

Comment:
Probably, better option is to install `sudo apt install build-essential libtool autotools-dev automake pkg-config`

Reply:
Seems like this was only included in the non-WSL instructions. Replacing!